psi4/driver/diatomic.py

```diff
@@ -149,12 +162,13 @@ def anharmonicity(rvals, energies, plot_fit='', mol = None):
     B0 = B - ae / 2.0
     r0 = np.sqrt(constants.h / (8.0 * np.pi**2 * mu * constants.c * B0))
     recheck = np.sqrt(constants.h / (8.0 * np.pi**2 * mu * constants.c * B))
-    r0 /= angstrom_to_meter;
-    recheck /= angstrom_to_meter;
+    r0 /= angstrom_to_meter
+    recheck /= angstrom_to_meter
```

Comment:
Hah!  You can tell I have a hard time switching between C++ and Python sometimes :)